RoadMap_Movie/RoadMap_Movie/Service/ImageService/Services/FileManagerService.swift

```diff
@@ -28,21 +28,18 @@ final class FileManagerService: FileManagerServiceProtocol {
     }()
 
     private let cacheLifeTime: TimeInterval = FileManagerConstants.cacheLifeTime
-    private var imagesMap = [String: UIImage]()
 
     // MARK: - Public methods
 
-    func getImageFromCache(url: String) -> UIImage? {
+    func getImageFromCache(url: String) -> Data? {
         guard let fileName = getFilePath(url: url),
               let info = try? FileManager.default.attributesOfItem(atPath: fileName),
               let modificationDate = info[FileAttributeKey.modificationDate] as? Date
         else { return nil }
         let lifeTime = Date().timeIntervalSince(modificationDate)
-        guard lifeTime <= cacheLifeTime,
-              let image = UIImage(contentsOfFile: fileName) else { return nil }
-        DispatchQueue.main.async {
-            self.imagesMap[url] = image
-        }
+        let fileNameURL = URL(filePath: fileName)
+        guard lifeTime <= cacheLifeTime else { return nil }
+        let image = try? Data(contentsOf: fileNameURL)
```

Comment:
```suggestion
        let image = try? Data(contentsOf: fileNameURL)
```
ошибку надо обрабатывать